### PR TITLE
new documentation for ingestion control page

### PR DESCRIPTION
**Pull request by dussault-antoine**
### What does this PR do?

moving ingestion control page v2 guide to the main section

### Motivation

Ingestion Control Page GA

### Preview

https://docs-staging.datadoghq.com/antoine.dussault/ingestion_control_page_ga/tracing/trace_ingestion/ingestion_controls/

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/tracing/trace_ingestion/ingestion_controls.md [hunk 1]
```diff
@@ -6,7 +6,8 @@ aliases:
     - /tracing/trace_ingestion/ingestion_control_page
     - /account_management/billing/usage_control_apm/
     - /tracing/app_analytics/
-description: "Learn how to control span ingestion"
+    - /guide/ingestion_control_page
```
Comment (kayayarai) [inline content/en/tracing/trace_ingestion/ingestion_controls.md:9]:
```suggestion
    - /tracing/guide/ingestion_control_page/
```

File: content/en/tracing/trace_ingestion/ingestion_controls.md [hunk 2]
```diff
@@ -21,121 +22,112 @@ further_reading:
...
 
-Ingestion controls affect what traces are sent by your applications to Datadog. Stats and metrics are always calculated based on all traces, and are not impacted by ingestion controls.
+Ingestion controls affect what traces are sent by your applications to Datadog. Stats and [metrics][1] are always calculated based on all traces, and are not impacted by ingestion controls.
 
-Many instrumented services send 100% of their traces to Datadog by default. The Datadog Agent does not drop or sample any spans by default at volumes of up to 50 traces per second. It is common for high-volume services or services that experience intermittent traffic to not send 100% of spans by default. This 50-traces-per-second default ingestion is based on Intelligent Retention and keeps diverse traces by default.
+The Ingestion Control page provides visibility at the Agent and tracing libraries level into the ingestion configuration of your applications and services. From the [ingestion control configuration page][2] , you can:
```
Comment (kayayarai) [inline content/en/tracing/trace_ingestion/ingestion_controls.md:27]:
```suggestion
The Ingestion Control page provides visibility at the Agent and tracing libraries level into the ingestion configuration of your applications and services. From the [ingestion control configuration page][2], you can:
```

File: content/en/tracing/trace_ingestion/ingestion_controls.md [hunk 3]
```diff
@@ -21,121 +22,112 @@ further_reading:
...
+- Investigate and act on potential ingestion configuration issues (limited CPU or RAM resources for the Agent).
```
Comment (kayayarai) [inline content/en/tracing/trace_ingestion/ingestion_controls.md:28]:
```suggestion
- Investigate and act on potential ingestion configuration issues, such as limited CPU or RAM resources for the Agent.
```

File: content/en/tracing/trace_ingestion/ingestion_controls.md [hunk 4]
```diff
@@ -21,121 +22,112 @@ further_reading:
...
+{{< img src="tracing/trace_indexing_and_ingestion/ingestion_control_page.png" style="width:100%;" alt="Ingestion Control Page Overview" >}}
 
-{{< img src="tracing/trace_indexing_and_ingestion/IngestionControls.png" style="width:100%;" alt="Retention Filters" >}}
+All metrics used in the page are based on live traffic data of the **past 1 hour**. Any Agent or library configuration change is reflected in the page.
 
-**Note:** If you are seeing numbers below 100% for Ingestion Rate, ensure you are using Agent 6.19+ or 7.19+ as these versions increased the default rate.
+## Summary across all environments
 
-In Datadog, on the [Ingestion Control page][1], you can see the following information:
+Get an overview of the total ingested data over the past hour, and an estimation of your monthly usage against your monthly allocation, calculated with the active APM intrastructure (hosts, Fargate tasks and serverless functions).
```
Comment (kayayarai) [inline content/en/tracing/trace_ingestion/ingestion_controls.md:38]:
```suggestion
Get an overview of the total ingested data over the past hour, and an estimation of your monthly usage against your monthly allocation, calculated with the active APM intrastructure (hosts, Fargate tasks, and serverless functions).
```

File: content/en/tracing/trace_ingestion/ingestion_controls.md [hunk 5]
```diff
@@ -21,121 +22,112 @@ further_reading:
...
 
-Root Service
-: The name of each service instrumented and sending traces to Datadog.
+If the monthly usage is under `100%`, the projected ingested data fits in your [monthly per-host allotment][3]. On the contrary, a quota over `100%` means that the montly ingestion volume is estimated to be over the monthly allocation.
```
Comment (kayayarai) [inline content/en/tracing/trace_ingestion/ingestion_controls.md:40]:
```suggestion
If the monthly usage is under `100%`, the projected ingested data fits in your [monthly per-host allotment][3]. A quota over `100%` means that the monthly ingestion volume is projected to be over your monthly allocation.
```

File: content/en/tracing/trace_ingestion/ingestion_controls.md [hunk 6]
```diff
@@ -21,121 +22,112 @@ further_reading:
...
+## Service table view
 
-Ingestion Rate
-: A percentage from 0 to 100% of how many of the spans that are produced by the service are being ingested by Datadog.Any number lower than 100% means some traces are not being ingested by Datadog from the Datadog Agent, and these traces are being dropped by the Datadog Agent after metrics and stats have been calculated.
+The service table contains information by service about the ingestion configuration:
 
-Ingestion Breakdown
-: A detailed breakdown of the destination of every trace generated by the service. See [Ingestion Breakdown](#ingestion-breakdown) for more information.
+Services
+: The name of each service sending traces to Datadog. The table contains root and non-root services that ingested data in the past one hour.
```
Comment (kayayarai) [inline content/en/tracing/trace_ingestion/ingestion_controls.md:47]:
```suggestion
: The name of each service sending traces to Datadog. The table contains root and non-root services for which data was ingested in the past one hour.
```

File: content/en/tracing/trace_ingestion/ingestion_controls.md [hunk 7]
```diff
@@ -21,121 +22,112 @@ further_reading:
...
 
-Dropped Spans
-: The percentage of incoming spans dropped by the Datadog Agent.If this percent is higher than 0%, the service can be configured by clicking anywhere on the service row.See [Change the Default Ingestion Rate](#change-the-default-ingestion-rate) for more information.
+Ingested Bytes/s
+: Average number of bytes per second ingested into Datadog for the service in the past one hour.
 
-Traces Ingested per Second
-: Average number of traces per second ingested into Datadog for the service over the selected time period.
+Downstream Bytes/s
+: Average number of bytes per second ingested, included the spans from downstream services for which the service is responsible of the ingestion. This includes the bytes of all descending child spans, as well as spans caught by the [Error sampler][4], the [Rare sampler][5] and the [App Analytics][6] mechanism.
```
Comment (kayayarai) [inline content/en/tracing/trace_ingestion/ingestion_controls.md:56]:
```suggestion
: Average number of bytes per second ingested, including spans from downstream services whose ingestion settings are set at this service. This includes the bytes of all descending child spans, as well as spans caught by the [Error sampler][4], the [Rare sampler][5], and the [App Analytics][6] mechanism.
```

File: content/en/tracing/trace_ingestion/ingestion_controls.md [hunk 8]
```diff
@@ -21,121 +22,112 @@ further_reading:
...
 
-**Note:** This may impact your bill if your total ingestion exceeds the included GBs. For more information, see the [APM Billing][4] page.
+The Traffic Breakdown column breaks down the destination of all traces originating from the service. It gives you an estimate of the share of traffic that is ingested and dropped, and for which reasons.
 
-### Ingestion breakdown
-
-The Ingestion Breakdown column breaks down the destination of all traces originating from the service. It can help you understand lower than expected ingestion rates and missing traces.
-
-{{< img src="tracing/trace_indexing_and_ingestion/IngestionBreakdown.png" style="width:100%;" alt="breakdown of traces ingested" >}}
+{{< img src="tracing/trace_indexing_and_ingestion/service_traffic_breakdown.png" style="width:100%;" alt="traffic breakdown of trace ingestion" >}}
```
Comment (kayayarai) [inline content/en/tracing/trace_ingestion/ingestion_controls.md:78]:
```suggestion
{{< img src="tracing/trace_indexing_and_ingestion/service_traffic_breakdown.png" style="width:100%;" alt="Traffic breakdown of trace ingestion" >}}
```

File: content/en/tracing/trace_ingestion/ingestion_controls.md [hunk 9]
```diff
@@ -21,121 +22,112 @@ further_reading:
...
-    2. When you change the default ingestion rate to less than 100%.
+    1. By default, the [Agent distributes an ingestion rate][7] to services depending on service traffic.
+    2. When the service is manually [configured][8] to ingest a certain percentage of traces at the tracing library level.
 
-- **Complete traces dropped by the tracer rate limiter** (orange): When you choose to [configure the ingestion rate of a service](#change-the-default-ingestion-rate), you explicitly define the ingestion rate that your service should have. However, as a protection mechanism, a rate limiter set to 100 traces per second by default is automatically enabled. To configure this rate limiter, [open a support ticket][5] to have someone help guide you through the process.
+- **Complete traces dropped by the tracer rate limiter** (orange): When you choose to manually set the service ingestion rate as a percentage with rules, a rate limiter is set to 100 traces per second by default is automatically enabled. See the [rate limiter][8] documentation to manually configure this rate.
```
Comment (kayayarai) [inline content/en/tracing/trace_ingestion/ingestion_controls.md:88]:
```suggestion
- **Complete traces dropped by the tracer rate limiter** (orange): When you choose to manually set the service ingestion rate as a percentage with rules, a rate limiter is automatically enabled, set to 100 traces per second by default. See the [rate limiter][8] documentation to manually configure this rate.
```

File: content/en/tracing/trace_ingestion/ingestion_controls.md [hunk 10]
```diff
@@ -21,121 +22,112 @@ further_reading:
...
 
-- **Traces dropped due to the Agent CPU limit** (red): The Agent has a configuration option allowing users to limit the usage of the CPU. After this limit is reached the Agent stops accepting traces from the tracers. Change the [Agent configuration][6] to configure how much CPU to allocate to the Agent.
+- **Traces dropped due to the agent CPU or RAM limit** (red): This mechanism may drop spans and create incomplete traces. To fix this, increase the CPU and memory allocation for the infrastructure that the Agent runs on.
```
Comment (kayayarai) [inline content/en/tracing/trace_ingestion/ingestion_controls.md:90]:
```suggestion
- **Traces dropped due to the Agent CPU or RAM limit** (red): This mechanism may drop spans and create incomplete traces. To fix this, increase the CPU and memory allocation for the infrastructure that the Agent runs on.
```

File: content/en/tracing/trace_ingestion/ingestion_controls.md [hunk 11]
```diff
@@ -21,121 +22,112 @@ further_reading:
...
 
-If you are seeing ingestion rates below 100% within Datadog and would like to send all your traces, enable Tracing without Limits by setting the environment variable as described above. If you have questions, contact [support][5].
+Explore the **Ingestion reasons breakdown** to see which mechanisms are responsible for your service ingestion. Each ingestion reason relates to one specific [ingestion mechanism][10]. After changing your service ingestion configuration, you can observe the increase or decrease of ingested bytes and spans in this timeseries graph based on the past hour of ingested data.
 
-{{< img src="tracing/trace_indexing_and_ingestion/VisualIndicator.png" style="width:100%;" alt="Root services not sending 100% of traces" >}}
+If most of your service ingestion is due to decisions taken by upstream services, investigate the detail of the top uptream decision makers. For instance, if your service is non-root, (meaning that it **never decides** to sample traces), observe all upstream services responsible for your non-root service ingestion. Configure upstream root services to reduce your overall ingestion volume.
```
Comment (kayayarai) [inline content/en/tracing/trace_ingestion/ingestion_controls.md:100]:
```suggestion
If most of your service ingestion is due to decisions taken by upstream services, investigate the detail of the top upstream decision makers. For example, if your service is non-root, (meaning that it **never decides** to sample traces), observe all upstream services responsible for your non-root service ingestion. Configure upstream root services to reduce your overall ingestion volume.
```

File: content/en/tracing/trace_ingestion/ingestion_controls.md [hunk 12]
```diff
@@ -21,121 +22,112 @@ further_reading:
...
 
+For further investigations, use the [APM Trace - Estimated Usage Dashboard][11], which provides global ingestion information as well as breakdown graphs by `service`, `env` and `ingestion reason`.
 
-## App Analytics to Tracing Without Limits
+### Configure the service ingestion rate
 
-Before October 20, 2020, Datadog offered App Analytics to index spans for performing analytics. While this is no longer the recommended setup configuration and is not needed to use [Trace Explorer][7], the legacy instructions are available within the [App Analytics][8] setup page.
+Click `Manage Ingestion Rate` to configure your service ingestion rate.
```
Comment (kayayarai) [inline content/en/tracing/trace_ingestion/ingestion_controls.md:106]:
```suggestion
Click **Manage Ingestion Rate** to configure your service ingestion rate.
```

File: content/en/tracing/trace_ingestion/ingestion_controls.md [hunk 13]
```diff
@@ -6,7 +6,8 @@ aliases:
     - /tracing/trace_ingestion/ingestion_control_page
     - /account_management/billing/usage_control_apm/
     - /tracing/app_analytics/
-description: "Learn how to control span ingestion"
+    - /guide/ingestion_control_page
+description: "Learn how to control Ingestion and Indexing rates with APM."
```
Comment (kayayarai) [inline content/en/tracing/trace_ingestion/ingestion_controls.md:10]:
```suggestion
description: "Learn how to control Ingestion rates with APM."
```
Right? Indexing is for retention?

File: content/en/tracing/trace_ingestion/ingestion_controls.md [hunk 14]
```diff
@@ -21,121 +22,112 @@ further_reading:
...
-: Amount of data ingested by Datadog over the selected time period.
+## Service table view
 
-Ingestion Rate
-: A percentage from 0 to 100% of how many of the spans that are produced by the service are being ingested by Datadog.Any number lower than 100% means some traces are not being ingested by Datadog from the Datadog Agent, and these traces are being dropped by the Datadog Agent after metrics and stats have been calculated.
+The service table contains information by service about the ingestion configuration:
 
-Ingestion Breakdown
-: A detailed breakdown of the destination of every trace generated by the service. See [Ingestion Breakdown](#ingestion-breakdown) for more information.
+Services
```
Comment (kayayarai) [inline content/en/tracing/trace_ingestion/ingestion_controls.md:46]:
```suggestion
Name
```

File: content/en/tracing/trace_ingestion/ingestion_controls.md [hunk 15]
```diff
@@ -21,121 +22,112 @@ further_reading:
...
 
-Dropped Spans
-: The percentage of incoming spans dropped by the Datadog Agent.If this percent is higher than 0%, the service can be configured by clicking anywhere on the service row.See [Change the Default Ingestion Rate](#change-the-default-ingestion-rate) for more information.
+Ingested Bytes/s
+: Average number of bytes per second ingested into Datadog for the service in the past one hour.
 
-Traces Ingested per Second
-: Average number of traces per second ingested into Datadog for the service over the selected time period.
+Downstream Bytes/s
+: Average number of bytes per second ingested, including spans from downstream services whose ingestion settings are set at this service. This includes the bytes of all descending child spans, as well as spans caught by the [Error sampler][4], the [Rare sampler][5], and the [App Analytics][6] mechanism.
```
Comment (dussault-antoine) [inline content/en/tracing/trace_ingestion/ingestion_controls.md:56]:
```suggestion
: Average number of bytes per second ingested, including spans from downstream services whose ingestion decisions are set at this service. This includes the bytes of all descending child spans, as well as spans caught by the [Error sampler][4], the [Rare sampler][5], and the [App Analytics][6] mechanism.
```

**Pull request merged**